resources/assets/less/bem/osu-page-header.less

```diff
@@ -45,6 +76,27 @@
     height: 165px;
   }
 
+  &--artist {
+    background-size: cover;
+    background-position: center;
+    height: 200px;
+    justify-content: flex-start;
+
+    font-weight: 100;
+    color: white;
+
+    &__subtitle {
```

Comment:
apparently there's even an faq entry for this http://getbem.com/faq/#block-modifier-affects-elements